frontend/app/src/App.vue

```diff
@@ -247,4 +247,27 @@ export default class App extends Vue {
     }
   }
 }
+
+::-webkit-scrollbar {
```

Comment:
Since we merged the pr with the scss file wouldn't just importing the scss file here have the same effect?

Reply:
Fixed in post-rebase PR.